blockchain/blockvalidator.go

```diff
@@ -57,6 +56,25 @@ var (
 	ErrBalance = errors.New("invalid balance")
 )
 
+// Start registers validators
+func (v *validator) Start(ctx context.Context) error {
+	raCtx, ok := protocol.GetRunActionsCtx(ctx)
```

Comment:
validators are supposed to be extracted from validateActionsContext in function validateActions, line 176